src/lib/core.js

```diff
@@ -17,6 +17,8 @@ module.exports = function (options) {
     creds.options.prefix = ledgerPrefix
     core.addClient(ledgerPrefix, new ilpCore.Client(Object.assign({}, creds.options, {
       debugReplyNotifications: config.features.debugReplyNotifications,
+      // disable connect() timeout
+      connectTimeout: 0,
```

Comment:
I think so. Also add it to the ledger plugin interface spec. If you're passing it to the ledger plugin no matter what type it is, and if we think it's a useful options for other plugins to have, it should be part of the standard options.